packages/adapter-react/README.md

```diff
@@ -53,17 +53,38 @@ The generator will create a custom React component and context inside `i18n-reac
 Wrap your application root with the `TypesafeI18n` component:
 
 ```jsx
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { detectLocale, navigatorDetector } from 'typesafe-i18n/detectors';
```

Comment:
please get rid of all semi-colons (`;`) to match the style of all other examples